modules/govuk_env_sync/files/govuk_env_sync.sh

```diff
@@ -713,6 +695,11 @@ else
   : "${url?"No storage url specified (pass -u option)"}"
   : "${path?"No storage path specified (pass -p option)"}"
 
+  if [[ "$dbms" == "postgresql" ]] && [ -z "${database_hostname+x}" ]; then
```

Comment:
Simplifies to
```suggestion
  if [[ "$dbms" == "postgresql" && -z "$database_hostname" ]]; then
```

Reply:
Great - thank you. That was the end result of me Googling and a lot of trial and error 😆

Reply:
Hmm I tried that out in https://github.com/alphagov/govuk-puppet/pull/11343/commits/d48360132cb203f2c488f2fcd727a39f1ed43379 and it brought back the error I faced earlier of: 

```
/usr/local/bin/govuk_env_sync.sh: line 698: database_hostname: unbound variable
```

I'm going to revert that change now, but still welcome suggestions.

Reply:
Oh right, sorry I didn't test my suggestion with `set -u`. Try `"${database_hostname:-}"`

Reply:
That works 👍  Thanks 🏅